packages/core/signal.mjs

```diff
@@ -86,6 +86,27 @@ export const tri2 = fastcat(isaw2, saw2);
 
 export const time = signal(id);
 
+/**
+ *  The mouse's x or y position value ranges from 0 to 1.
+ *
+ * @return {Pattern}
+ * @example
+ * n(mousex.segment(4).range(0,7)).scale("C:minor")
+ * n(mousey.segment(4).range(0,7)).scale("C:minor")
+ *
+ */
+let _mouseY = 0,
+  _mouseX = 0;
+if (typeof window !== 'undefined') {
+  document.onmousemove = (e) => {
+    _mouseY = e.clientY / document.body.clientHeight;
+    _mouseX = e.clientX / document.body.clientWidth;
+  };
+}
+
+export const mousey = signal(() => _mouseY);
```

Comment:
i never thought about using `signal` for outside references.. so far i've used the `ref` function but maybe that isn't even needed